app/views/registrations/register.html.erb

```diff
@@ -32,7 +32,7 @@
     <% end %>
   <% else %>
     <% user_may_register = current_user.cannot_register_for_competition_reasons(@competition).length == 0 %>
-    <% if @competition.should_render_register_v2?(current_user) %>
+    <% if @competition.should_render_register_v2?(current_user) && !@registration.competing_status_cancelled? %>
```

Comment:
this doesn't render the registration panel at all now when a registration is cancelled, which is not correct

Reply:
Ah of course - didn't realise the check was happening in the React component. I think I've moved it to the correct place now?